packages/design-system/src/templates/Card.scss

```
@@ -0,0 +1,28 @@
+.card-template {
+	background-repeat: no-repeat;
+	background-position: center;
+	background-size: cover;
+
+	background-image: url('../images/datachaos.png'), linear-gradient(134deg, #19426c, #2c1f56);
+	overflow: auto;
+
+	:global(.layout__main) {
+		background: none;
+		padding: 5rem;
+	}
+
+	&__card {
+		width: 100vw;
+		margin: 0 auto;
+		max-width: 65rem;
```

width: 100vw + max-width: 65 rem --> width: 65rem
```suggestion
		width: 65rem;
```